Econ490_PS1.do

```diff
@@ -0,0 +1,141 @@
+*Week 4 Assignment
+*Oghenefegor Omorojor
+
+cd "/Users/user/Box Sync/Week4/04_assignment/data"
```

Comment:
might consider delete this line as it conflicts with global

Reply:
A redo has been committed